avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroAnnotationIntrospector.java

```diff
@@ -113,6 +124,10 @@ public Object findSerializer(Annotated a) {
         if (a.hasAnnotation(Stringable.class)) {
             return ToStringSerializer.class;
         }
+        AvroEncode ann = _findAnnotation(a, AvroEncode.class);
+        if (ann != null) {
+            return CustomEncodingSerializer.class;
```

Comment:
@baharclerode Ok, filed & fixed https://github.com/FasterXML/jackson-databind/issues/1581 -- it was regression from 2.8, but wasn't covered by unit tests (ironically enough I had added some tests for `AnnotationIntrospectorPair` in 2.9... but not for this method).